### PR TITLE
 Fix ResizeImage and data transform for SSD 

### PR DESCRIPTION
**Pull request by qingqing01**
Fix http://newicafe.baidu.com/issue/75838627/show?cid=5&spaceId=23205&from=email&noticeStatistics=22629182

### CODE REVIEW

**Review by SunAhong1993 — APPROVED** [review]

File: PaddleCV/PaddleDetection/docs/MODEL_ZOO.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ results of image size 608/416/320 above.
 
 | Backbone     | Size | Image/gpu | Lr schd | Box AP | Download  |
 | :----------- | :--: | :-----: | :-----: | :----: | :-------: |
-| MobileNet v1 | 300  |    32   |   120e  |  73.2  | [model](https://paddlemodels.bj.bcebos.com/object_detection/ssd_mobilenet_v1_voc.tar) |
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/MODEL_ZOO.md:90]: 73.2和73.13是同一个模型吗？

File: PaddleCV/PaddleDetection/ppdet/data/transform/operators.py [hunk 2]
```diff
@@ -172,15 +177,19 @@ def __call__(self, sample, context=None):
...
-                ],
-                dtype=np.float32)
+                [resize_h, resize_w, im_scale], dtype=np.float32)
         else:
             im_scale_x = float(selected_size) / float(im_shape[1])
             im_scale_y = float(selected_size) / float(im_shape[0])
+
+            resize_w = selected_size
+            resize_h = selected_size
+
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/ppdet/data/transform/operators.py:192]: Error will occur if use_cv2=true in SSD

File: PaddleCV/PaddleDetection/docs/MODEL_ZOO.md (hunk 1 above)
Comment (qingqing01) [reply PaddleCV/PaddleDetection/docs/MODEL_ZOO.md:90]: Yeah, I test https://paddlemodels.bj.bcebos.com/object_detection/ssd_mobilenet_v1_voc.tar.   原来的测试程序有bug，数据预处里有Flip.

File: PaddleCV/PaddleDetection/ppdet/data/transform/operators.py (hunk 2 above)
Comment (qingqing01) [reply PaddleCV/PaddleDetection/ppdet/data/transform/operators.py:192]: 我这边训练没问题:

```
pies
I0729 04:11:20.729091 118597 build_strategy.cc:342] SeqOnlyAllReduceOps:0, num_trainers:1
2019-07-29 04:11:31,675-INFO: iter: 0, lr: 0.001000, 'loss': 39.689362, time: 0.000, eta: 0:00:13
2019-07-29 04:11:38,920-INFO: iter: 20, lr: 0.001000, 'loss': 7.889134, time: 0.970, eta: 7:32:07
2019-07-29 04:11:51,120-INFO: iter: 40, lr: 0.001000, 'loss': 6.802613, time: 0.618, eta: 4:48:11
2019-07-29 04:12:02,505-INFO: iter: 60, lr: 0.001000, 'loss': 6.741658, time: 0.565, eta: 4:23:09
2019-07-29 04:12:16,199-INFO: iter: 80, lr: 0.001000, 'loss': 5.858238, time: 0.679, eta: 5:15:53
```

**Review by jerrywgz — APPROVED** [review]
LGTM

**Pull request merged**